django/utils/datastructures.py

```diff
@@ -131,18 +131,21 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
-        """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
-        """
+    def _getlist(self, key, default=None):
```

Comment:
By the looks of code that uses `MultiValueDict` a `getlist` is expected to be mutable. `QueryDict` actually contains  a `_mutable` flag which is set True on copy wich should be reflected to `getlist` in `QueryDict`.